tools/uninstall.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+# vim: tabstop=4 softtabstop=4 shiftwidth=4 noexpandtab
+###############################################################################
+# This is the Linux installation script for the King Phisher Client and
+# Server on supported distributions.
+#
+# Project Home Page: https://github.com/securestate/king-phisher/
+# Authors:
+#   Erik Daguerre
+#
+# Supported Linux Distributions:
+#   Linux Flavor  | Client | Server |
+#   --------------|--------|--------|
+#   BackBox		  |   yes  |   yes  |
+#   CentOS		  |   no   |   yes  |
+#   Debian		  |   yes  |   yes  |
+#   Fedora		  |   yes  |   yes  |
+#   Kali		  |   yes  |   yes  |
+#   Ubuntu		  |   yes  |   yes  |
+#
+###############################################################################
+
+E_USAGE=64
+E_SOFTWARE=70
+E_NOTROOT=87
+FILE_NAME="$(dirname $(readlink -e $0) 2>/dev/null)/$(basename $0)"
+GIT_CLONE_URL="https://github.com/securestate/king-phisher.git"
+if [ -z "$KING_PHISHER_DIR" ]; then
```

Comment:
This and the 2 lines below it don't seem necessary, the directory can be found using the two methods you have implemented towards the end.